README.md

```diff
@@ -1,14 +1,14 @@
-[![Build Status](https://travis-ci.org/CODAIT/max-workshop-app-nodejs.svg?branch=master)](https://travis-ci.org/CODAIT/max-workshop-app-nodejs)
+[![Build Status](https://travis-ci.org/IBM/max-workshop-app-nodejs.svg?branch=master)](https://travis-ci.org/IBM/max-workshop-app-nodejs)
 
 # MAX Object Detector Web App (Lite) - NodeJS
 
 A slimmed down version of the [MAX Object Detector Web App](https://github.com/IBM/MAX-Object-Detector-Web-App) for use in MAX workshops.
 
 This repository's default branch, `initial`, contains **TODOs** meant to be completed while following the
-[MAX Developer Tutorial](https://github.com/CODAIT/max-developer-tutorial).
+[MAX Developer Tutorial](https://github.com/IBM/max-developer-tutorial).
 A working version of the app without the **TODOs** can be found on the `solution` branch.
 
-> This is the node version of this Web App, you can also check out the [Python version](https://github.com/CODAIT/max-workshop-app-python)
+> This is the node version of this Web App, you can also check out the [Python version](https://github.com/IBM/max-workshop-app-python)
```

Comment:
repo name change workshop -> tutorial
`> This is the node version of this Web App, you can also check out the [Python version](https://github.com/IBM/max-tutorial-app-python)`